pkgs/top-level/all-packages.nix

```diff
@@ -22032,7 +22032,7 @@ with pkgs;
 
   libglibutil = callPackage ../development/libraries/libglibutil { };
 
-  libgnome-keyring = callPackage ../development/libraries/libgnome-keyring { };
+  libgnome-keyring = gnome.libgnome-keyring;
```

Comment:
> Those libraries actually have the same API.

Hmm I tend to think maybe we should use a `lib.warn` alias instead? Maybe there are other differences we are not aware of?

Reply:
Well, one difference is that the newer version marks everything deprecated.

Reply:
> Well, one difference is that the newer version marks everything deprecated so it can cause problems if some project has deprecations disabled. But I do not expect that to happen – most consumers do not even include the headers, they just dlopen the library. And I would expect sizeable portion to be unused – the library has been deprecated for over 10 years.
> 
> Here is the abidiff for peace of mind:

OK Great it's nice to get confidence that this won't break anything, but still, the reason to add a `lib.warn` is to distribute the update - that's our role as a distribution IMO (same arguments as in the ["How to introduce a breaking version bump" discourse thread](https://discourse.nixos.org/t/how-to-introduce-a-breaking-version-bump/46746)).

Reply:
I do not think there is a consensus on `lib.warn` yet so I decided to go with status quo for `aliases.nix` and not add it here.